app/TracingStrategyAssets.ts

```diff
@@ -1,9 +1,11 @@
 import { useTranslation } from 'react-i18next';
 
-import { Images } from './assets';
+import { Icons, Images } from './assets';
 import { isGPS } from './COVIDSafePathsConfig';
 
-export function useAssets(): Record<string, string> {
+type Asset = string | ((name: string) => string) | null;
```

Comment:
Thanks for making this type a bit more accurate! Though it does jump out at me as a bit board. What do you think about splitting up the various asset types and such that the functions return values a more explicit?
something like: 
```
export const useStrategyTranslation = (): Record<string, string> => { ...

export const useStragteyImage = (): Record<string, Image> => {

export const useStrategyContent = (): Record<string, (name: string) => string> => {
```
?

Reply:
I like this! Is it cool if I do this on a different PR?